staging/src/k8s.io/metrics/pkg/apis/metrics/register.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 // GroupName is the group name use in this package
-const GroupName = "metrics"
+const GroupName = "resource-metrics.metrics.k8s.io"
```

Comment:
Why not having just `metrics.k8s.io`. metrics.metrics sounds weird to me.

Reply:
We already have custom-metrics.metrics.k8s.io, so it just seemed strange to use it as both a "namespace" for groups and a group itself.  This would make it resource-metrics.metrics and custom-metrics.metrics, so they match.

Reply:
can we have:
`metrics.k8s.io` for resource metrics
`custom.metrics.k8s.io` for custom metrics?